src/main/java/CodeCatalyst.java

```diff
@@ -0,0 +1,130 @@
+import java.util.Scanner;
+
+public class CodeCatalyst {
+    private static final int MAX_TASKS = 100;
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        Task[] tasks = new Task[MAX_TASKS];
+        int taskCount = 0;
+
+        printGreeting();
+
+        while (true) {
+            String input = scanner.nextLine();
+            printDivider();
+
+            if (input.equals("bye")) {
+                printGoodbye();
+                break;
+            } else if (input.equals("list")) {
+                printTaskList(tasks, taskCount);
+            } else if (input.startsWith("mark ")) {
+                handleTaskStatusChange(tasks, taskCount, input, true);
+            } else if (input.startsWith("unmark ")) {
+                handleTaskStatusChange(tasks, taskCount, input, false);
+            } else if (input.startsWith("todo ")) {
+                taskCount = addTask(tasks, taskCount, new Todo(input.substring(5)));
+            } else if (input.startsWith("deadline ")) {
+                taskCount = addDeadlineTask(tasks, taskCount, input);
+            } else if (input.startsWith("event ")) {
+                taskCount = addEventTask(tasks, taskCount, input);
+            } else {
+                taskCount = addTask(tasks, taskCount, new Task(input));
+            }
+            printDivider();
+        }
+        scanner.close();
+    }
+
+    private static void printDivider() {
+        System.out.println("        ________________________________________________________\n");
```

Comment:
Consider refactoring the divider string into a constant instead.
```suggestion
        private static final String SEPARATOR = "        _____________________________________\n";
        System.out.println(SEPARATOR);
```